.github/pull_request_template.md

```diff
@@ -1,39 +1,46 @@
 ## Description
 
-A detailed explanation of the changes in your PR.
+<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
 
-## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
+All three items *must be* checked:
+<!-- Remove conflicting items that won't be checked. -->
 
-If any of these don't apply, please comment below.
+- [ ] CI results are inspected
+- [ ] CHANGELOG is updated
+- [ ] I believe CHANGELOG update is not needed
+- [ ] Documentation PR is created and linked above
+- [ ] Documentation is not needed
 
-## Testing Performed
+## Testing
 
-### Here I tell how I validated my change
+### Automated testing
 
-TODO(replace-me)  
+There *must be* at least one item and all items *must be* checked:
+<!-- Remove item(s) that don't apply and won't be checked. -->
+
+- [ ] added unit tests
+- [ ] added e2e tests
+- [ ] added regression tests
+- [ ] added compatibility tests
+- [ ] modified existing tests
+- [ ] **no automated tests**
```

Comment:
Nit: let's have this item in a consistent voice with the ones above. Sorry, did not consider myself when suggested it.

```suggestion
- [ ] **I did not contribute to automated tests**
```